packages/flutter/test/material/tooltip_test.dart

```diff
@@ -713,10 +713,7 @@ void main() {
           child: const Icon(Icons.add),
```

Comment:
The changes in this file look like they may not be related to the change. We should not change existing tests as they are if they are not part of the change. That could introduce regressions.